app/javascript/react/components/Button/Button.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+import { HeaderButton } from "./Button.style";
```

Comment:
Let's agree to always follow:
```suggestion
import * as S from "./Button.style";
```

Then:
<S.Button>...